latest/ug/nodes/hybrid-nodes-cluster-update.adoc

```diff
@@ -2,32 +2,31 @@ include::../attributes.txt[]
 
-You can do any of the following actions to the EKS Hybrid Nodes in a cluster:
+You can do any of the following actions to the EKS Hybrid Nodes networking configuration in a cluster:
 
 * Add remote node network configuration to enable EKS Hybrid Nodes in an existing cluster.
-* Add, change, or remove the remote node networks. You can remove individual CIDR ranges but you can't remove all remote node networks.
+* Add, change, or remove the remote node networks. Remove all remote node network CIDR ranges to disable EKS Hybrid Nodes in an existing cluster.
 * Add, change, or remove the optional remote pod networks. 
 * Remove all of the remote pod networks.
-* Remove remote node network configuration to disable EKS Hybrid Nodes in an existing cluster.
 
 The following examples enable EKS Hybrid Nodes on an existing cluster and include the optional remote pod network.
 
 [#hybrid-nodes-cluster-enable-prep]
 == Prerequisites
 
-* The <<hybrid-nodes-prereqs>> completed. Before you create your hybrid nodes-enabled cluster, you must have your on-premises node and optionally pod CIDRs identified, your VPC and subnets created according to the EKS requirements, and hybrid nodes requirements, and your security group with inbound rules for your on-premises and optionally pod CIDRs. For more information on these prerequisites, see <<hybrid-nodes-networking>>.
+* The <<hybrid-nodes-prereqs>> completed. Before you create your hybrid nodes-enabled cluster, you must have your on-premises node and optionally pod CIDRs identified, your VPC and subnets created according to the EKS and hybrid nodes requirements, and your security group with inbound rules for your on-premises node and optionally pod CIDRs. For more information on these prerequisites, see <<hybrid-nodes-networking>>.
 * The latest version of the {aws} Command Line Interface ({aws} CLI) installed and configured on your device. To check your current version, use `aws --version`. Package managers such yum, apt-get, or Homebrew for macOS are often several versions behind the latest version of the {aws} CLI. To install the latest version, see link:cli/latest/userguide/getting-started-install.html[Installing or updating to the last version of the {aws} CLI,type="documentation"] and link:cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config[Configuring settings for the {aws} CLI,type="documentation"] in the {aws} Command Line Interface User Guide.
 * An link:IAM/latest/UserGuide/id_roles#iam-term-principal[IAM principal,type="documentation"] with permissions to create IAM roles and attach policies, and create and describe EKS clusters
```

Comment:
This last bullet is not accurate. The only permission the IAM role needs is update cluster.